app/controllers/api_controller/chargeback.rb

```diff
@@ -0,0 +1,66 @@
+class ApiController
+  module Chargeback
+    def chargebacks_query_resource(object)
+      object.tags
+    end
+
+    # Request
+    # {
+    #   "action":"assign",
+    #
+    #   "data": [
+    #     {
+    #         "object_type": "tenant",
+    #         "object_id": 1
+    #     }
+    #   ]
+    # }
```

Comment:
The "data" key is not consistent with the other signatures, maybe look at tags assign/unassign, signature is via the "resources" for multiple, and "resource" for single. resources is handled by the API and would call chargebacks_assign_resource for each (i.e. no need for first, etc.). Thanks.